Sources/CSRF/CSRFFormFieldTag.swift

```diff
@@ -0,0 +1,23 @@
+import Vapor
+import Leaf
+
+public struct CSRFFormFieldTag: LeafTag {
+
+	public init() {}
+
+	public func render(_ ctx: LeafContext) throws -> LeafData {
+
+		struct CSRFFormFieldTagError: Error {}
```

Comment:
Prefer to put things like this in an extension.

Additionally, since you're `throw`ing this below, it should have some documentation (and be `public`?). An enum with a good name could suffice.